ginga/misc/plugins/LineProfile.py

```diff
@@ -0,0 +1,186 @@
+from ginga import GingaPlugin
+from ginga.misc import Widgets, Plot
+
+import numpy as np
+
+class LineProfile(GingaPlugin.LocalPlugin):
+
+    def __init__(self, fv, fitsimage):
+        super(LineProfile, self).__init__(fv, fitsimage)
+
+        self.layertag = 'lineprofile-canvas'
```

Comment:
You don't need a this kind of variable unless you are creating a canvas that gets inserted and removed over the channel widget.